app/views/targets/new.html.haml

```diff
@@ -0,0 +1,85 @@
+#googleMap
+%script{:src => "http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"}
+:javascript
+  function myMap() {
+    var mapProp= {
+        center:new google.maps.LatLng(-34.906268,-56.186120),
+        zoom:15,
+    };
+
+    var map=new google.maps.Map(document.getElementById("googleMap"),mapProp);
+
+    var marker = new google.maps.Marker({
+              position: new google.maps.LatLng(-34.906268,-56.186120),
+              map: map,
+              icon: {
+            		url: "/assets/marker.svg"
+            	}
+            });
+
+    map.addListener('click', function(e) {
+        placeMarker(e.latLng, map);
+    });
+
+    function placeMarker(location) {
+      if (marker == null) {
+        marker = new google.maps.Marker({
+        position: location,
+        map: map
+        });
+      } else {
+         marker.setPosition(location);
+      }
+      $('#latitude').val(location.lat);
+      $('#longitude').val(location.lng);
+    }
+  }
+  function loadLocation() {
+    $('#latitude').val(-34.906268);
+    $('#longitude').val(-56.186120);
+  }
+%script{:src => "https://maps.googleapis.com/maps/api/js?key=AIzaSyAiW1JcyQxgKFXNbSLy19mciRsUqSneuE4&callback=myMap"}
+= form_for @target, url: {action: "create"}, html: {id:"rectangle-3"} do |f|
+  #rectangle-2
+    %p#create-target CREATE TARGET
+  = image_tag image_path("group.svg"), |
+    id: "group"                        |
+  %p#create-new-target CREATE NEW TARGET
+  %p#specify-area-lenght SPECIFY AREA LENGHT
+  #rectangle
+    = f.select(:length,[['100 m',100],['200 m',200],['500 m',500],['1 km', 1000]],{},id: "length")
+  %p#target-title TARGET TITLE
+  #rectangle-copy
+    = f.text_field(:title, id:"choose-a-title-for-y", value:"Choose a title for your target")
+    %br/
+  %p#select-a-topic SELECT A TOPIC
+  #rectangle-copy-2
+    %p#what-do-you-want-to What do you want to talk about?
+  #rectangle-5
+    %ul#topics
+      %li#Football Football
```

Comment:
Look into an alternative for this that iterates on an array of strings to generate this list. How could you fetch dynamically?